source/_docs/pantheon-yml.md

```diff
@@ -36,6 +36,32 @@ protected_web_paths:
 * You may not be able to protect files or paths with special characters
 * Wait a few seconds for changes to take effect
 
+### Enforce HTTPS + HSTS
```

Comment:
The markup in this section is broken. It's not rendering as a list.

Reply:
thanks, @joshkoenig! this is fixed in my most recent push
check it out on the pre-built here https://httpsnong-static-docs.pantheonsite.io/docs/pantheon-yml/#enforce-https-+-hsts